redis/Cargo.toml

```diff
@@ -10,4 +10,4 @@ repository = "https://github.com/khuey/bb8"
 bb8 = { path = "../" }
 futures = "0.1"
 tokio = "0.1"
-redis = "0.11"
+redis = "0.12"
```

Comment:
This requires bumping the version of bb8-redis as well to 0.4.0-pre, since the new redis is semver incompatible.

Reply:
Yep, I think you are right. Is this a problem?

Reply:
No, it's not a problem.